data.json

```diff
@@ -95,6 +95,14 @@
     "image": "https://partners.npmjs.com/weekly/weekly30/socks-1100x.jpg",
     "tags": ["clothing"]
   },
+  {
+    "name": "OpenBazaar",
+    "difficulty": "medium",
+    "description": "Help improve our code and earn a limited edition shirt.",
+    "reference": "https://twitter.com/openbazaar/status/1053016648347717632",
+    "image": "https://pbs.twimg.com/media/Dp0RD4yXcAASC6r.jpg:orig",
```

Comment:
I found these 3 shirts:
- https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7/openbazaar-t-shirt
- https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7/original-openbazaar-t-shirt
- https://openbazaar.com/store/QmcUDmZK8PsPYWw5FRHKNZFjszm2K6e68BQSTpnJYUsML7/openbazaar-decentralize-together-t-shirt

It will probaly be one of these, we may want to use one of these pictures as the swag image.